blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -165,17 +188,16 @@ export class RunDetailsPipeline extends Component {
                          * some steps.
                          */
                         if (event.pipeline_step_stage_id !== this.mergedConfig.node && parallel) {
-                            // console.log('nodes fetching via sse triggered');
-                            delete this.mergedConfig.node;
-                            fetchNodes({ ...this.mergedConfig, refetch });
+                            // console.log('nodes fetching via ssePipeline triggered');
+                            refetchNodes();
                         } else {
-                            // console.log('only steps fetching via sse triggered');
-                            fetchSteps({ ...this.mergedConfig, refetch });
+                            // console.log('only steps fetching via ssePipeline triggered');
+                            setTimeout(fetchSteps({ ...this.mergedConfig, refetch }), 500);
```

Comment:
I was trying/adopted it from the PR of the rerun button, but it did not really fixed the glitches that I can sometime see.